flink/v1.16/flink/src/main/java/org/apache/iceberg/flink/source/enumerator/ContinuousSplitPlannerImpl.java

```diff
@@ -62,11 +64,26 @@ public ContinuousSplitPlannerImpl(Table table, ScanContext scanContext, String t
                 "iceberg-plan-worker-pool-" + threadName, scanContext.planParallelism());
   }
 
+  public ContinuousSplitPlannerImpl(
```

Comment:
we can probably remove or directly change the other constructor. this is an `@Internal` class. not user facing.